modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/GiveRatingFragment.kt

```diff
@@ -73,18 +73,26 @@ class GiveRatingFragment : BaseDialogFragment() {
                         onDismiss = ::dismiss,
                         setRating = viewModel::setRating,
                         submitRating = {
-                            viewModel.submitRating(
-                                context = context,
-                                onSuccess = {
-                                    Toast.makeText(context, "TODO: Submit rating", Toast.LENGTH_LONG).show()
-                                    dismiss()
-                                },
-                            )
+                            coroutineScope.launch {
+                                viewModel.submitRating(
+                                    podcastUuid = podcastUuid,
+                                    context = context,
+                                    onSuccess = {
+                                        Toast.makeText(context, getString(LR.string.thank_you_for_rating), Toast.LENGTH_LONG).show()
```

Comment:
This is just an opinion but I think this would be nicer at a Snackbar instead of a Toast message. I don't think they need an action but having it in the app rather than float above the app is nicer. 

> Alternatives to using toasts
> If your app is in the foreground, consider using a [snackbar](https://material.io/components/snackbars) instead of using a toast. Snackbars include user-actionable options, which can provide a better app experience.

https://developer.android.com/guide/topics/ui/notifiers/toasts#:~:text=If%20your%20app%20is%20in,action%2C%20use%20a%20notification%20instead.